contracts/contracts/sbtc-withdrawal.clar

```diff
@@ -189,12 +194,14 @@
 )
 ;; Reject multiple withdrawal requests
 (define-public (complete-withdrawals (withdrawals (list 600 
-                                     {request-id: uint, 
-                                     status: bool, 
-                                     signer-bitmap: uint, 
-                                     bitcoin-txid: (optional (buff 32)), 
-                                     output-index: (optional uint), 
-                                     fee: (optional uint)})))
+                                      {request-id: uint, 
+                                      status: bool, 
+                                      signer-bitmap: uint, 
+                                      bitcoin-txid: (optional (buff 32)), 
+                                      output-index: (optional uint), 
+                                      fee: (optional uint)}))
+                                     (burn-height uint)
+                                     (burn-hash (buff 32)))
```

Comment:
Can we do this for `accept-withdrawal-request` as well?

Reply:
Completed tests & was just looking at this & it requires a bit more refactor that I thought going in. As a result of using a helper to loop through multiple deposits & withdrawals I see two options moving forward, neither of which are great:

1. We update *individual* deposits & withdrawals with a burn-height & burn-hash field (which I don't love since multiple deposits & withdrawals might now have different burn-heights & burn-hashes)
2. We don't include burn-height/burn-hash in individual deposits & withdrawals but instead print them both from the sbtc-deposit & sbtc-withdrawal contracts instead of in the confirmed sbtc-registry contract (which I don't love since we currently have all print statements in sbtc-registry)

Reply:
Yeah, I definitely prefer option (1). And there is also option (3): each withdrawal tuple has a `burn-height` and `burn-hash` field. The trade off there is that we need to submit more data when making the contract call, making things more expensive. This is a slight modification on option (1) but I like (1) more.

But I think we can keep the updated signature as is and clarity things in the docs: only make `complete-withdrawals` and `complete-deposits` contract calls where the requests were completed in the same bitcoin block.

Reply:
Updated individual deposits to take in the burn-hash & the burn-height fields, can you confirm that the deposit print statements are what you expect? If so I'll continue with refactoring withdrawal.

Reply:
Yup, this looks good.